src/components/Card/Card.test.tsx

```
@@ -0,0 +1,52 @@
+      expect(subtitle.className).toContain(`text-${action}-`);
+      unmount();
+    });
+  });
+
+  it('should render icon', () => {
+    render(<CardActivesResults {...baseProps} />);
+    expect(screen.getByTestId('icon')).toBeInTheDocument();
+  });
+
+  it('should accept and apply custom className', () => {
+    render(<CardActivesResults {...baseProps} className="custom-class" data-testId='test-class'/>);
```

ESLint

[error] Replace `<CardActivesResults·{...baseProps}·className="custom-class"·data-testId='test-class'/>` with `⏎······<CardActivesResults⏎········{...baseProps}⏎········className="custom-class"⏎········data-testId="test-class"⏎······/>⏎····`

(prettier/prettier)